doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -100,7 +100,9 @@ It is not necessary for Glassfish to be running before you execute the Dataverse
 
 Please note that you must run Glassfish in an English locale. If you are using something like ``LANG=de_DE.UTF-8``, ingest of tabular data will fail with the message "RoundRoutines:decimal separator no in right place".
 
-Also note that Glassfish may utilize more than the default number of file descriptors, especially when running batch jobs such as harvesting. We have increased ours by adding ulimit -n 32768 to our glassfish init script. If there is a better way to do this for CentOS 7 boxes, please let us know. 
+Also note that Glassfish may utilize more than the default number of file descriptors, especially when running batch jobs such as harvesting. We have increased ours by adding ulimit -n 32768 to our glassfish init script. On operating systems which use systemd such as RHEL or CentOS 7, file descriptor limits may be increased by adding a line like LimitNOFILE=32768 to the systemd unit file. You may adjust the file descriptor limits on running processes by using the prlimit utility:
+
+	$ sudo prlimit -p pid -n 32768:32768
```

Comment:
This example command is not formatted like other examples on the page (see Line 87), missing a `#` character to format correctly.

Reply:
Also, not sure if it required, but inline command/setting/code examples can/should be formatted using double backquote characters ```` which could apply to ulimit -n 32768 and LimitNOFILE=32768.